### PR TITLE
Changes on property applications

### LINKED ISSUE #1212: changes on applications and their compatible graphs
Let's revisit all the applications and their compatible graphs: 
- in general, make applications defined on flattened graphs, rather than property graphs. e.g., we don't need a SSSP on property graph. 
- reserve specific apps defined on property graph, e.g., u-i lpa;
- make the projection step implicit in Python. 
- for applications with many variants, make the most general one as its default. e.g., lpa -> cdlp maybe ? 
- whether Pregel API on property or not is to be decided.

### PR DESCRIPTION
**Pull request by lidongze0629**
## What do these changes do?

- [x] Remove SSSP and BFS on property graph.
- [x] cdlp -> lpa,  lpa -> lpa_u2i
- [ ] make the projection step implicit in Python.
- [x] reserve specific apps defined on property graph, e.g., u-i lpa;
- [x] test and update doc of analytical engine

## Related issue number

Fixes #1212 
Links #1121

### CODE REVIEW

File: python/graphscope/tests/unittest/test_session.py [hunk 1]
```diff
@@ -53,72 +54,6 @@ def local_config_file():
         yield json_path
 
 
-# load property graph to specified session
```
Comment (lidongze0629) [inline python/graphscope/tests/unittest/test_session.py:56]: Replace with builtin dataset.

File: python/graphscope/tests/unittest/test_context.py [hunk 2]
```diff
@@ -68,49 +68,13 @@ def test_simple_context_to_vineyard_dataframe(
     assert out is not None
 
 
-def test_property_context_to_numpy(property_context):
```
Comment (lidongze0629) [inline python/graphscope/tests/unittest/test_context.py:71]: move `test_property_context` into `test_udf_app.py`

File: python/graphscope/tests/unittest/test_context.py [hunk 3]
```diff
@@ -68,49 +68,13 @@ def test_simple_context_to_vineyard_dataframe(
...
-    out = property_context.to_numpy("v:v0.dist")
-    assert out.shape == (40521,)
-    out = property_context.to_numpy("r:v1.dist_1")
-    assert out.shape == (40786,)
-
-
-def test_property_context_to_dataframe(property_context):
```
Comment (lidongze0629) [inline python/graphscope/tests/unittest/test_context.py:78]: ditto.

File: python/graphscope/tests/unittest/test_context.py [hunk 4]
```diff
@@ -68,49 +68,13 @@ def test_simple_context_to_vineyard_dataframe(
...
-
-
-def test_property_context_to_vineyard_dataframe(graphscope_session, property_context):
-    out = property_context.to_vineyard_dataframe(
-        {"id": "v:v0.id", "data": "v:v0.dist", "result": "r:v0.dist_0"}
-    )
-    assert out is not None
-
-
-def test_add_column(arrow_property_graph, property_context):
```
Comment (lidongze0629) [inline python/graphscope/tests/unittest/test_context.py:106]: ditto.

File: python/graphscope/tests/unittest/test_graph.py [hunk 5]
```diff
@@ -190,49 +182,6 @@ def test_load_only_from_efile(
     assert sorted(r_out1) == sorted(r_out2)
 
 
-def test_graph_to_numpy(arrow_property_graph):
```
Comment (lidongze0629) [inline python/graphscope/tests/unittest/test_graph.py:193]: Already test in [test_lazy.py](https://github.com/alibaba/GraphScope/blob/main/python/graphscope/tests/unittest/test_lazy.py#L189) with  `lazy` mode

File: python/graphscope/tests/unittest/test_lazy.py [hunk 6]
```diff
@@ -73,21 +72,6 @@ def teacher_group_e(data_dir=os.path.expandvars("${GS_TEST_DIR}/property_graph")
     return Loader("%s/teacher_group.e" % data_dir, header_row=True, delimiter=",")
 
 
-def arrow_property_graph(graphscope_session):
```
Comment (lidongze0629) [inline python/graphscope/tests/unittest/test_lazy.py:76]: Replace with builtin dataset

File: docs/analytics_engine.rst [hunk 7]
```diff
@@ -16,25 +16,28 @@ Built-in Algorithms
...
-    g = graphscope.g()
-    # algorithms defined on property graph can be invoked directly.
-    result = lpa(g)
+    graphscope.set_option(show_log=True)
+
+    # load property graph
+    g = load_p2p_network()
 
-    # some other algorithms may only support evaluate on simple graph
+    # Algorithms only support evaluate on simple graph
```
Comment (yecol) [inline docs/analytics_engine.rst:33]: Many algorithms only ....

**Review by yecol — APPROVED** [review]

File: python/graphscope/framework/app.py [hunk 8]
```diff
@@ -312,7 +312,7 @@ class name.
         self._session.dag.add_op(self._op)
 
     def __repr__(self):
-        s = f"graphscope.App <type: {self._app_assets.type}, algorithm: {self._app_assets.algo}"
+        s = f"graphscope.App <type: {self._app_assets.type}, algorithm: {self._app_assets.algo} "
```
Comment (sighingnow) [inline python/graphscope/framework/app.py:315]: Why we need a trailing space ` ` here?

**Review by sighingnow — APPROVED** [review]

File: docs/analytics_engine.rst [hunk 9]
```diff
@@ -330,38 +326,35 @@ To run your own algorithms, you may trigger it in place where you defined it.
...
-    g = sess.g()
+    g = load_p2p_network()
 
     # load my algorithm
     my_app = SSSP_Pregel()
 
     # run my algorithm over a graph and get the result.
     # Here the `src` is correspondent to the `context.get_config(b"src")`
-    ret = my_app(g, src="0")
+    ret = my_app(g, src="6")
```
Comment (siyuan0322) [inline docs/analytics_engine.rst:338]: Is this a string "6"? or a int 6.

File: docs/analytics_engine.rst [hunk 10]
```diff
@@ -330,38 +326,35 @@ To run your own algorithms, you may trigger it in place where you defined it.
...
-    g = graphscope.g()
+    from graphscope.framework.app import load_app
 
     # load my algorithm from a gar package
-    my_app = load_app('file:///var/graphscope/udf/my_sssp_pregel.gar')
+    my_app = load_app("/tmp/my_sssp_pregel.gar")
 
     # run my algorithm over a graph and get the result.
-    ret = my_app(g, src="0")
+    ret = my_app(g, src="6")
```
Comment (siyuan0322) [inline docs/analytics_engine.rst:357]: ditto. Have you checked the result's correctness?

**Review by siyuan0322 — APPROVED** [review]
Overall LGTM.

File: docs/analytics_engine.rst (hunk 9 above)
Comment (lidongze0629) [reply docs/analytics_engine.rst:338]: String is needed here.

https://github.com/alibaba/GraphScope/blob/main/analytical_engine/core/app/pregel/pregel_context.h#L65

File: docs/analytics_engine.rst (hunk 10 above)
Comment (lidongze0629) [reply docs/analytics_engine.rst:357]: Check result in [UDF CI Test](https://github.com/alibaba/GraphScope/blob/main/python/graphscope/tests/unittest/test_udf_app.py#L1009)

File: python/graphscope/framework/app.py (hunk 8 above)
Comment (lidongze0629) [reply python/graphscope/framework/app.py:315]: There is a string `add` operation in `__repr__` function.
```
def __repr__(self):
    s = f"graphscope.App <type: {self._app_assets.type}, algorithm: {self._app_assets.algo} "
    s += f"bounded_graph: {str(self._graph)}>"
    return s
```

**Pull request merged**